src/Codeception/Module/Db.php

```diff
@@ -729,7 +729,7 @@ public function _loadDump(string $databaseKey = null, array $databaseConfig = nu
         $databaseKey = empty($databaseKey) ?  self::DEFAULT_DATABASE : $databaseKey;
         $databaseConfig = empty($databaseConfig) ?  $this->config : $databaseConfig;
 
-        if ($databaseConfig['populator']) {
+        if (array_key_exists('populator', $databaseConfig) && !empty($databaseConfig['populator'])) {
```

Comment:
`array_key_exists` is unnecesary, because `!empty` handles the case then key doesn't exists.

Reply:
The logic is unaltered, but a notice is triggered. See https://github.com/Codeception/module-db/actions/runs/3557783372/jobs/5979012692#step:7:115